src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
You are linking a SCSS file here. It should be a CSS file. The browser does not understand SCSS syntax. You need to compile your SCSS to CSS and then link the CSS file here.